include/exadg/utilities/print_solver_results.h

```diff
@@ -191,6 +191,20 @@ print_wall_time(dealii::ConditionalOStream const & pcout, double const wall_time
   // clang-format on
 }
 
+inline void
+print_wall_time_with_mass_iter(dealii::ConditionalOStream const & pcout,
```

Comment:
according to another comment, think about whether this function can be removed and whether an existing function can be used.